configure_check_conditions.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<p>Change phrasing or add another section, explaining that several conditions can be added...</p><p> </p><p>btw - General note - would be best to start with "Add a Condition" (with a subtitle)</p><p>and then it would be natural to state that every check in a suite can have any number of conditions (zero included)</p><p> </p><p>And then only later do the "Remove or Edit a Condition" (also add a subtitle for it)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>